WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -150,5 +151,30 @@ public static void ShellExec(string cmd, bool waitForExit = true)
 				}
 			}
 		}
+
+		public static bool IsFileTypeAssociated(string fileExtension)
+		{
+			// Source article: https://edi.wang/post/2019/3/4/read-and-write-windows-registry-in-net-core
+
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				throw new InvalidOperationException("Operation only supported on windows.");
+			}
```

Comment:
Shouldn't this return false instead?

Reply:
Here I warn the coder that he is using the method incorrectly so that's why I use exception.